sharepoint/sharepoint-ps/sharepoint-online/Set-SPOBuiltInFontPackageSettings.md

```diff
@@ -0,0 +1,36 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spobuiltinfontpackagesettings
+applicable: SharePoint Online
+title: Set-SPOBuiltInFontPackageSettings
+schema: 2.0.0
+author: yixianpu
+ms.author: yixianpu
+ms.reviewer:
+---
+
+# Set-SPOBuiltInFontPackageSettings
+
+## SYNOPSIS
+
+Sets settings of built-in font packages.
+ 
+## SYNTAX
+
+```powershell
+Set-SPOBuiltInFontPackageSettings [-HideBuiltInFontPackages] <Boolean>
+```
+## DESCRIPTION
+
+Use this cmdlet to set settings of built-in font packages.
+
+## EXAMPLE
+
+Set-SPOBuiltInFontPackageSettings -HideBuiltInFontPackages $true
+
+## PARAMETERS
+
+### -HideBuiltInFontPackages
+
+Hide built in font packages from SharePoint sites if this value is true.
```

Comment:
>Hide built [](http://example.com/codeflow?start=0&length=10)

please work with your PM on this documentation. e.g., from reading this, it's not clear what "hiding" means/what effect it has etc